src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/AbstractParenPadCheck.java

```diff
@@ -110,17 +110,16 @@ else if (option == PadOption.SPACE && !hasWhitespaceAfter
     protected void processRight(DetailAST ast) {
         final int before = ast.getColumnNo() - 1;
         if (before >= 0) {
-            final String line = getLines()[ast.getLineNo() - 1];
-            final int[] codePoints = line.codePoints().toArray();
+            final int[] line = getLineCodePoints(ast.getLineNo() - 1);
             final boolean hasPrecedingWhitespace =
-                    CommonUtil.isCodePointWhitespace(codePoints, before);
+                    CommonUtil.isCodePointWhitespace(line, before);
```

Comment:
@MUzairS15 please send PR to move this utility to your new `CodePointUtil` class.

Reply:
@nick-mancuso Will, that be ok, If I send PR after all check updates are merged?

Reply:
Sure, please add comment in issue so that we do not forget.

Reply:
@MUzairS15 ping

Reply:
added https://github.com/checkstyle/checkstyle/issues/10924#issuecomment-1068710441

Reply:
Resolved.